apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.Data/SpannerCommandTextBuilder.cs

```diff
@@ -0,0 +1,195 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Linq;
+using Google.Api.Gax;
+
+// ReSharper disable UnusedParameter.Local
+
+namespace Google.Cloud.Spanner
+{
+    /// <summary>
+    /// </summary>
+    public sealed class SpannerCommandTextBuilder
+    {
+        private static readonly string s_insertCommand = "INSERT";
```

Comment:
const instead of static readonly? (Then just InsertCommand rather than s_insertCommand.)